src/PortingEmbed/features/AccountHolder/AccountHolderForm.tsx

```
@@ -29,9 +49,20 @@ export function AccountHolderForm({
     birthday: porting.birthday || undefined,
   })
 
+  const [validationErrors, setValidationErrors] = useState<string | null>(null)
+
   function handleSave() {
-    onSubmit(formData)
+    try {
+      schema.parse(formData)
+      onSubmit(formData)
+    } catch (err) {
+      if (err instanceof ZodError) {
+        const errorsToDisplay = err.errors.map((e) => e.message)
+        setValidationErrors(`${errorsToDisplay.join('. ')}.`)
+      }
+    }
```

Compared to the previous version, the `catch` would now swallow all errors thrown by `onSubmit`. Is this intentional, or do we need to re-throw it?

---

right no we should rethrow 👌